security-response.md

```diff
@@ -0,0 +1,107 @@
+# Security Response Guidelines
+
+Security vulnerabilities should be handled quickly and sometimes privately. The
+primary goal of this process is to reduce the total time users are vulnerable to
+publicly known exploits.
+
+The OpenTelemetry (OTel) Technical Committee (TC) and relevant repo maintainers,
+supported by tooling provided by the Sig-Security, are responsible for
+responding to the incident organizing the entire response including internal
+communication and external disclosure.
+
+## Supported Versions
+
+The OTel project provides community support only for the last minor
```

Comment:
```suggestion
The OTel project provides community support only for the latest overall minor
```
To make clear it's not the respective latest minor for each individual major version but only the single overall most recent one.